i18n/validate.py

```diff
@@ -171,6 +175,10 @@ def report_problems(filename, problems):
 
 
 class Validate(Runner):
+    """
+    Implements Runner interface
+    """
```

Comment:
Thanks for ensuring the class has a docstring, but this docstring doesn't explain anything about the class.  Let's change it to:  "The `validate` command for checking that .po files have no errors."